RULES.md

```diff
@@ -36,6 +36,8 @@ Rules are declared in the [`Notice` module](https://github.com/MobilityData/gtfs
 | [E032](#E032) | `calendar.txt` `end_date` is before `start_date` |
 | [E033](#E033) | `route_id` not found |
 | [E035](#E035) | `agency_id` not found |
+| [E036](#E035) | Stop with invalid parent `location_type` |
```

Comment:
Suggest slight wording change for rule to be more consistent with others:

```suggestion
| [E036](#E035) | Invalid parent `location_type` for stop |
```

Reply:
Oh, the error code needs to be changed to `E036` too (I just fixed that in my suggested change above.